docs/UserGuide.md

```diff
@@ -60,23 +60,23 @@ You can also generate a PDF payslip for your employees to refer to.
-| `clear`     | `clear`                                                                                  | NA                                                                                                      |
-| `job`       | `job n/NAME d/DURATION`                                                                  | `job n/Fix HDB Lock d/1`                                                                                |
-| `jobfind`   | `jobfind NAME`                                                                           | `jobfind Painting`                                                                                      |
-| `joblist`   | `joblist`                                                                                | NA                                                                                                      |
-| `jobdelete` | `jobdelete JOB_INDEX`                                                                    | `jobdelete 3`                                                                                           |
-| `assign`    | `assign JOB_INDEX i/PERSON_INDEX [i/PERSON_INDEX]...​`                                   | `assign 2 i/1`                                                                                          |
-| `mark`      | `mark JOB_INDEX`                                                                         | `mark 2`                                                                                                |
-| `pay`       | `pay JOB_INDEX y/`                                                                       | `pay 2 y/`                                                                                              |
-| `exit`      | `exit`                                                                                   | NA                                                                                                      |
-| `help`      | `help`                                                                                   | NA                                                                                                      |
+| Command        | Format                                                                                        | Examples                                                                                                |
+|----------------|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| `add`          | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS r/RATE [t/TAG]...​`                              | `add n/Nicole Tan p/99338558 e/nicole@stffhub.org  a/1 Tech Drive, S138572 r/37.50 t/Hardware t/Senior` |
```

Comment:
should this also be `personadd` for consistency?

Reply:
Good point, I had the same consideration. Currently, the command for adding a job is `job`, so `person` or `personadd` (which also implies changing `job` to `jobadd`) seemed natural, but this is a command that would potentially be used a lot? `personadd`/`jobadd` seemed like a mouthful to type out in comparison to `add`, which is why I let it be.

I guess the added consistency is more relevant though; I'll push a commit to account for this

Reply:
Changes made:
`add` -> `personadd`
`job` -> `add`